### ISSUE TITLE
Some logging improvements

### ISSUE THREAD

**Issue by 36000** [yeatmanlab/pyAFQ#914]
Some simple fixes:
Label tqdm bars;
print before doing SYN fitting that that is what you are doing
print before doing CSD fitting that that is what you are doing

Have an option for the CLI to print output to a file. In this case, make sure the TQDM bars do not take up a bunch of lines

**Comment by pierre-nedelec** [comment]
When using pyAFQ through the Python API (e.g. with `ParticipantAFQ.export_all()`), how do we get log to file?

**Comment by 36000** [comment]
You can redirect stdout and/or stderr to a log file. One of these solutions could work:
https://stackoverflow.com/questions/4675728/redirect-stdout-to-a-file-in-python

**Comment by pierre-nedelec** [comment]
Should probably consider tqdm mininterval with various values for streaming vs logging. 
https://github.com/tqdm/tqdm#monitoring-thread-intervals-and-miniters
Include in https://github.com/yeatmanlab/pyAFQ/pull/920?

**Comment by arokem** [comment]
Yeah - this is really annoying. Here's another potential approach: https://stackoverflow.com/a/75059930

**Comment by arokem** [comment]
And yes, could you please add one of these approaches to #920? That would be great.
